giskard/integrations/avid.py

```diff
@@ -0,0 +1,98 @@
+import datetime
+
+import giskard
+
+from ..core.errors import GiskardImportError
+from ..datasets import Dataset
+from ..models.base import BaseModel
+from ..scanner.issues import Issue
+
+try:
+    from avidtools.datamodels.components import (
+        Artifact,
+        ArtifactTypeEnum,
+        AvidTaxonomy,
+        ClassEnum,
+        Detection,
+        Impact,
+        LangValue,
+        LifecycleEnum,
+        MethodEnum,
+        Metric,
+        Problemtype,
+        Reference,
+        SepEnum,
+        TypeEnum,
+    )
+    from avidtools.datamodels.report import Affects, Report
+except ImportError:
+    raise GiskardImportError("Avidtools is not installed. Please install it with `pip install avidtools`.")
+
+
+def create_report_from_issue(issue: Issue, model: BaseModel, dataset: Dataset = None) -> Report:
+    """Create a report from an issue.
+
+    Parameters
+    ----------
+    issue : Issue
+        Issue to create a report from.
+    model : BaseModel
+        Model that was evaluated.
+    dataset : Dataset, optional
+        Dataset that was used for evaluation, by default ``None``.
+
+    Returns
+    -------
+    Report
+        AVID Report created from the issue.
+    """
+    # Prepare artifacts
+    artifacts = [Artifact(type=ArtifactTypeEnum.model, name=model.meta.name)]
+    if dataset:
+        artifacts.append(Artifact(type=ArtifactTypeEnum.dataset, name=dataset.meta.name))
+
+    affects = Affects(
+        developer=[],
+        deployer=[],
```

Comment:
not sure how important these fields are but we can allow them to be filled by including some kwargs in:
```py
def to_avid(self, filename=None, **kwargs):
   ...
```

Reply:
I would rather avoid that because in principle they could be easily filled on the generated report, like
```
report.developer = [AvidObject...]
```